cypress/support/lpa-commands.js

```diff
@@ -11,3 +11,11 @@ Cypress.Commands.add("createLpa", (donorId) =>
       cy.postToApi(`/api/v1/donors/${donorId}/lpas`, lpa).its("body")
     )
 );
+
+Cypress.Commands.add("createEpa", (donorId) =>
+  cy
+    .fixture("epa/minimal.json")
+    .then((lpa) =>
+      cy.postToApi(`/api/v1/donors/${donorId}/epas`, lpa).its("body")
```

Comment:
Definitely not a blocker, but in case you happen to change something else, this variable should be `epa`
```suggestion
    .then((epa) =>
      cy.postToApi(`/api/v1/donors/${donorId}/epas`, epa).its("body")
```